### PR TITLE
Refresh parameter test on amm liquidity

### LINKED ISSUE #4601: Property-based testing of AMM calc failed in CI
## Describe the bug

In CI, @warner saw this  https://github.com/Agoric/agoric-sdk/runs/5251436118?check_suite_focus=true
It's the result of a new property-based test checked in with #4519.

At the last minute, I removed maximum limit on some values. #4600 reinstated the caps. This issue is for investigating whether the failure represents an actual issue in the code.

## Expected behavior

Property based testing uses random inputs. The version run in CI should probably use a stable seed, but only after sufficient runs to be sure there aren't uncovered corner cases.

## Platform Environment

This ran in CI. Other than reproducing the random seed, it's reproducible locally.

## Screenshots

Screenshot 2022-02-18 at 09-51-11 merge bundle_vat-admin devices add waitForBundlecap() · Agoric_agoric-sdk@9f531c5

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes: #4601
 
## Description

#4519  added a test with some failures. #4600 temporarily repaired it. This is a better fix that increases the range of cases we test, with a narrow exclusion for ill-behaved parameters.

### Security Considerations

No actual security issues. The economic security issue is that someone might use the new ability to add liquidity to the AMM in an infelicitous circumstance (the pools have a small amount of collateral to start) and end up giving more money to the arbitrageurs than they intended.

### Documentation Considerations

When we explain `makeAddLiquidityAtRateInvitation`, we need to explain where the edge cases are.

### Testing Considerations

This PR repairs a property-based test that by its nature doesn't run with the same values every time, so it can give different results on different runs. The exclusions mentioned above are intended to skip over values with problematic results.

### CODE REVIEW

File: packages/run-protocol/test/amm/vpool-xyk-amm/test-amm-calc.js [hunk 1]
```diff
@@ -33,6 +47,13 @@ test('balancesToReachRatio calculations are to spec', t => {
         giveY: arbGiveY,
       }),
       ({ poolX, poolY, giveX, giveY }) => {
+        const kBefore = Number(multiply(poolX.value, poolY.value));
+        const poolRatioAfter = Math.max(
+          Number(giveX.value + poolX.value) / Number(giveY.value + poolY.value),
+          Number(giveY.value + poolY.value) / Number(giveX.value + poolX.value),
+        );
+        // skip cases where the target ratio is more than K at the start
+        fc.pre(poolRatioAfter < kBefore);
```
Comment (dckc) [inline packages/run-protocol/test/amm/vpool-xyk-amm/test-amm-calc.js]: does this `fc.pre` check subsume the `imbalancedRequest(` check below?

**Review by dckc — APPROVED** [review]
looks good!

File: packages/run-protocol/test/amm/vpool-xyk-amm/test-amm-calc.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/run-protocol/test/amm/vpool-xyk-amm/test-amm-calc.js]: It's exactly the same calculation.  thanks.
